.github/BOTMETA.yml

```diff
@@ -135,12 +135,16 @@ files:
   $modules/cloud/linode/: $team_linode
   $modules/cloud/lxd/: hnakamur
   $modules/cloud/memset/: glitchcrab
-  $modules/cloud/misc/ovirt.py: &virt
+  $modules/cloud/misc/ovirt.py: &ovirt
     ignored: skvidal vincentvdk
```

Comment:
This should probably not be retained ?

Reply:
I left it, because it was there in virt, but OK I can remove